openfisca_core/tools/test_runner.py

```diff
@@ -212,14 +213,28 @@ def check_variable(self, variable_name, expected_value, period, entity_index = N
 
         actual_value = self.simulation.calculate(variable_name, period)
 
+        absolute_error_margin = self.test.get('absolute_error_margin')
+        variable_absolute_error_margin = (
+            absolute_error_margin.get(variable_name, absolute_error_margin.get("default"))
```

Comment:
Why call it default and not just absolute error margin?

Reply:
It is the default of the absolute error margin